packages/expo-file-system/CHANGELOG.md

```diff
@@ -4,8 +4,11 @@
 
 ### 🛠 Breaking changes
 
+- `FileSystem.downloadAsync` and `FileSystem.DownloadResumable` now works in the background - they won't reject when the user backgrounds the application. ([#7380](https://github.com/expo/expo/pull/7380) [@lukmccall](https://github.com/lukmccall))
```

Comment:
```suggestion
- `FileSystem.downloadAsync` and `FileSystem.DownloadResumable` works in the background from now on - they won't reject when the application is moved to the background. ([#7380](https://github.com/expo/expo/pull/7380) [@lukmccall](https://github.com/lukmccall))
```

Maybe there's even a better noun to express this `foreground` 🔄 `background` transition process 🤔